### DISCUSSION TITLE
Server can't find some AA audio units

### DISCUSSION THREAD

**Discussion by MikyBass** [apohl79/audiogridder#623] [Help] [answered]
Hi mates. First of all, thanks for your amazing software.

I'm in trouble with some Acustica Audio plugins. They all worked fine until yesterday, but after installing a new demo, some of then don't work anymore: Pink4, Sand3 and Snow. The rest is ok, but the server doesn't scan this ones. I've tried Wipe cache & rescan several times. Nothins happens.

They work fine in Logic, so I suppose the installation is right. I've uninstalled all the AA plugin, even Aquarius. Have uninstalled AudioGridder too, to make a clean start. Reinstalled all from zero, first AA then AG. Try an AU validation in Terminal. Nothing works. Logic can find it, but server does not.

Any idea, please? I'm finishing a mixing work and I NEED it to work!

**Comment by apohl79** [comment, marked as answer]
You can reset AG by deleting the .audiogridder directory in your home folder. Open the terminal app and run "rm -rf ~/.audiogridder".

**Reply by MikyBass** [reply]
It works!!! Thank you very much, it's the second time you save me. :-)
